optimum/onnxruntime/modeling_ort.py

```diff
@@ -217,6 +217,9 @@ def shared_attributes_init(
         AutoConfig.register(self.model_type, AutoConfig)
         self.auto_model_class.register(AutoConfig, self.__class__)
 
+        # Define the pattern here to avoid recomputing it everytime.
+        self.output_shape_inference_pattern = re.compile(r"([a-zA-Z_]+)|([0-9]+)|([+-/*])|([\(\)])")
```

Comment:
What is `([\(\)])`?

Reply:
I'm not sure I understand why, couldn't find an output named with parenthesis, do you have an example? You could add a comment linking to it, it would be helpful.

Reply:
It's to cover the case where we want to do something like: `(sequence_length + past_length) / 2`. 
Currently we dont have a use for it but it could be needed at some point so added it here.